src/lib/client/utils/loadClerkJsScript.ts

```diff
@@ -0,0 +1,41 @@
+import { parsePublishableKey } from '@clerk/shared/keys'
+import { loadScript } from '@clerk/shared/loadScript'
+
+import type { ClerkInitOptions } from './types'
+
+const FAILED_TO_LOAD_ERROR = 'Clerk: Failed to load Clerk'
+
+export async function loadClerkJsScript(opts: ClerkInitOptions) {
+	const { publishableKey } = opts
+
+	if (!publishableKey) {
+		throw new Error('ClerkService requires a publishableKey')
+	}
+
+	return loadScript(clerkJsScriptUrl(opts), {
+		async: true,
+		crossOrigin: 'anonymous',
+		beforeLoad: applyClerkJsScriptAttributes(opts),
+	}).catch(() => {
+		throw new Error(FAILED_TO_LOAD_ERROR)
+	})
+}
```

Comment:
Got the idea [here](https://github.com/clerk/javascript/blob/8488bac8238e02d68f4c1025168b40a6101e29cc/packages/react/src/utils/loadClerkJsScript.ts#L18)

Reply:
we are planning to export these utilities in `@clerk/shared`